pkg/ensurance/collector/cadvisor/cadvisor_linux.go

```diff
@@ -25,69 +24,59 @@ import (
 	"github.com/gocrane/crane/pkg/utils"
 )
 
-const (
-	cadvisorCollectorName = "cadvisor"
-)
-
 func init() {
-	registerMetrics(cadvisorCollectorName, []types.MetricName{types.MetricNameContainerCpuTotalUsage, types.MetricNameContainerSchedRunQueueTime}, NewCadvisor)
+	//registerMetrics(cadvisorCollectorName, []types.MetricName{types.MetricNameContainerCpuTotalUsage, types.MetricNameContainerSchedRunQueueTime}, NewCadvisor)
```

Comment:
remove the init() function if it is not needed

Reply:
done~